src/theory/quantifiers/sygus/cegis_unif.h

```diff
@@ -261,6 +262,28 @@ class CegisUnif : public Cegis
                                   std::vector<Node>& candidate_values,
                                   bool satisfiedRl,
                                   std::vector<Node>& lems) override;
+  /** communicate condition values to solution building utility
+   *
+   * for each unification candidate and for each strategy point associated with
+   * it, set in d_sygus_unif the condition values (unif_cvalues) for respective
+   * condition enumerators (unif_cenums)
+   */
+  void setConditions(std::map<Node, std::vector<Node>>& unif_cenums,
+                     std::map<Node, std::vector<Node>>& unif_cvalues,
+                     std::vector<Node>& lems);
+  /** set values of condition enumerators based on current enumerator assignment
+   *
+   * This function also takes generates inter-enumerator symmetry breaking for
+   * return values, such that their model values are ordered by size
+   *
+   * returns true if symmetry breaking lemmas were generated for the return
+   * value enumerators, false otherwise
+   */
+  bool setEnumValues(const std::vector<Node>& enums,
```

Comment:
Document the arguments of this function. It's not clear what unif_cenums and unif_values are. (I'm assuming map from unif candidate to a list for conditionals/return values?)